### PR TITLE
Add Python version to ISSUE_TEMPLATE.md

### PR DESCRIPTION
**Pull request by CapCap**
Short Description: 

Fixes:
- 
- 
-

### CODE REVIEW

**Comment by douglascamata** [comment]
👍

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -10,3 +10,4 @@
 ### Other Information
 OS: 
 Git Commit: (run 'git log -n 1 --pretty=format:"%H"' and paste it here)
+Python Version: (run 'python-V' and paste it here)
```
Comment (DayBr3ak) [inline .github/ISSUE_TEMPLATE.md]: `python -V` is missing a space

**Comment by TheSavior** [comment]
:+1:

**Comment by douglascamata** [comment]
👍

**Pull request merged**